src/Shared/CertificateGeneration/CertificateManager.cs

```diff
@@ -621,7 +679,8 @@ private static void RemoveCertificateFromKeyChain(string keyChain, X509Certifica
             bool trust = false,
             bool includePrivateKey = false,
             string password = null,
-            string subjectOverride = null)
+            string subjectOverride = null,
+            bool isInteractive = true)
```

Comment:
Why did this default to true? Wouldn't it be safer to assume non-interactivity when not explicitly told otherwise?

Reply:
You are right, `isInteractive = false` is a bit safer, however there's only one place where we change the default and we have test coverage for the scenario, so I'm going to keep it like it is to avoid spinning another build.